src/test/ui-fulldeps/ast_stmt_expr_attr.rs

```diff
@@ -18,7 +18,7 @@ use syntax::parse::new_parser_from_source_str;
 use syntax::parse::parser::Parser;
```

Comment:
Shouldn't this be a normal ui test?

Reply:
AFAIK, anything depending on compiler crates has to be a fulldeps test.

Reply:
Yeah, what I wonder tho if this test could not be expressed as a series of UI tests instead -- the structure seems to be just ordinary compile-pass / compile-fail wrt. "reject" and "check". Using `cfg(FALSE)` it seems to me we can migrate this test to use that instead and privatize some stuff in the process.